docs/conceptual/user-guide.rst

```diff
@@ -0,0 +1,170 @@
+.. meta::
+   :description: A library that can be loaded by ROCr to print the AMDGPU wavefront states
+   :keywords: ROCdebug-agent, ROCm, library, tool, rocr
+
+.. _user-guide:
+
+===========================
+ROCdebug-agent user guide
```

Comment:
```suggestion
ROC Debug Agent user guide
```

Reply:
We should try to use the proper name as noted on the What is ROCm page.

Reply:
Done